Sources/Request/Request/FormParam/Form.Data.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  File.swift
+//  
+//
+//  Created by brennobemoura on 16/11/20.
+//
+
+import Foundation
+
+public extension Form {
+    struct Data: FormParam {
+        private let data: Foundation.Data
+        private let fileName: String
+        private let mime: String
+
+        // fileName: image.jpg
+        public init(_ fileName: String, _ mediaType: MediaType, _ data: Foundation.Data) {
```

Comment:
I feel like the `data` parameter should come before the mime type. Also I think some labels would be nice:
```suggestion
        public init(_ data: Foundation.Data, named fileName: String, withType mediaType: MediaType) {
```